javatests/arcs/core/storage/StoreEndpointFake.kt

```diff
@@ -15,6 +15,8 @@ class StoreEndpointFake<Data : CrdtData, Op : CrdtOperation, T>:
     private var callbacks = mutableListOf<ProxyCallback<Data, Op, T>>()
     private var proxyMessages = mutableListOf<ProxyMessage<Data, Op, T>>()
 
+    var proxyMessageReturn = true
```

Comment:
This is odd... What's the point of this variable?

Reply:
I needed a way to get this to return `false` in one specific test, so I just went with this approach. (https://github.com/PolymerLabs/arcs/pull/4871/files#diff-2c7ef3d39394fb5b6b5f91bd00d2ccdbR101)